api/index.md

```diff
@@ -5,13 +5,570 @@ _navPath: toc.html
+   <td>
+   Contains numeric types that complement the numeric primitives (such as Byte, Double, and Int32) that are available in .NET Core.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Reflection.html">System.Reflection</a></td>
+   <td>
+   Contains types that retrieve information at runtime about assemblies, modules, members, parameters, and other entities in managed code by examining their metadata. These types also can be used to manipulate instances of loaded types, for example to hook up events or to invoke methods. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Reflection.Context.html">System.Reflection.Context</a></td>
+   <td>
+   Contains  single class that can be used to create a customized reflection context.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Reflection.Emit.html">System.Reflection.Emit</a></td>
+   <td>
+   Contains classes that can emit metadata and Microsoft intermediate language (MSIL) and optionally generate a PE file on disk. The primary clients of these classes are script engines and compilers.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Reflection.Metadata.html">System.Reflection.Metadata</a></td>
+   <td>
+   Contains a class with single extension method that retrieves raw metadata from an assembly. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Resources.html">System.Resources</a></td>
+   <td>
+   Contains classes that allow developers to create, store, and retreive culture-specific string resources used in an application. One of the most important classes of the <a href="http://dotnet.github.io/api/System.Resources.html">System.Resources</a> namespace is the <a href="http://dotnet.github.io/api/System.Resources.ResourceManager.html">ResourceManager</a> class.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Runtime.html">System.Runtime</a></td>
+   <td>
+   Contains types that control the operation of the runtime's garbage collector.   
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Runtime.CompilerServices.html">System.Runtime.CompilerServices</a></td>
+   <td>
+   Provides functionality for compiler writers who use managed code to specify attributes in metadata that affect the run-time behavior of the runtime.
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Runtime.ExceptionServices.html">System.Runtime.ExceptionServices</a></td>
+   <td>
+   Contains a single class to represent an exception whose state is captured at a certain point in code. 
+   </td>
+</tr>
+<tr>
+   <td><a href="http://dotnet.github.io/api/System.Runtime.InteropServices.html">System.Runtime.InteropServices</a></td>
+   <td>
+   Provides a wide variety of members that support COM interop and platform invoke services on Windows platforms. 
```

Comment:
I think InteropServices types are used for PInvoke on Unix too, not just on Windows.